README.md

```diff
@@ -1 +1,27 @@
-# wireless-regdb
\ No newline at end of file
+wireless-regdb
+==============
+
+[![License MirOS](https://img.shields.io/badge/license-MirOS-green.svg)](https://github.com/nyantec/wireless-regdb/blob/master/LICENSE)
+[![Crates.io](https://img.shields.io/crates/v/wireless-regdb.svg)](https://crates.io/crates/wireless-regdb)
+[![doc.rs](https://docs.rs/wireless-regdb/badge.svg)](https://docs.rs/wireless-regdb)
+
+Read a wireless regdb, and convert it to a binary firmware file.
+
+Inspired by the python code from kernel.googlesource.com/pub/scm/linux/kernel/git/sforshee/wireless-regdb
```

Comment:
This should be a link? Or at least https if not a link?

Reply:
fixed